tests/raftstore/cluster.rs

```diff
@@ -100,15 +106,25 @@ impl<T: Simulator> Cluster<T> {
     }
 
     pub fn start(&mut self) {
+        self.start_with_strategy(vec![Strategy::Default]);
+    }
+
+    pub fn start_with_strategy(&mut self, strategy: Vec<Strategy>) {
         for engine in &self.dbs {
-            let node_id = self.sim.wl().run_node(0, self.cfg.clone(), engine.clone());
+            let node_id = self.sim
+                              .wl()
+                              .run_node(0, self.cfg.clone(), engine.clone(), strategy.clone());
```

Comment:
we are in a for loop, if we use strategy, compiler will complain "use of moved value"